### DISCUSSION TITLE
Setwarp cannot be named in Chinese

### DISCUSSION THREAD

**Discussion by VIQingNiao** [EssentialsX/Essentials#4884] [Q&A]
![essx setwarp](https://user-images.githubusercontent.com/46963763/162601446-cf2e4acc-66c5-41f6-a387-96487ee8bb6c.png)

A file with the same name will appear, which makes it impossible to create a warp，The file supports Chinese naming

**Comment by molor** [comment]
The same problem also with home names and happens with symbols from any language other than English, e.g. Cyrillic etc. And there's no way to configure this..
